src/clarity-angular/popover/dropdown/dropdown.module.ts

```diff
@@ -9,12 +9,14 @@ import { CommonModule } from "@angular/common";
 import { ClrCommonPopoverModule } from "../common/popover.module";
 import { ClrIconModule } from "../../icon/icon.module";
 import { DROPDOWN_DIRECTIVES } from "./index";
+import { ClrConditonalModule } from "../../utils/conditional/conditional.module";
 
 @NgModule({
     imports: [
         CommonModule,
         ClrCommonPopoverModule,
-        ClrIconModule
+        ClrIconModule,
+        ClrConditonalModule
```

Comment:
Same thing, but for this one I'm 100% sure. We don't want to import `ClrConditonalModule`, we want to export it. That way consumers importing `ClrDropdownModule` don't need to import anything else, they get all the dependencies automatically. That's the point of dependencies management. 😛